src/abaqus/Amplitude/ActuatorAmplitude.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, name: str, timeSpan: SymbolicConstant = STEP):
         RangeError
         """
         super().__init__()
-        pass
+        ...
```

Comment:
I think we should remove the ellipse where there is already a statement, right?

Reply:
Did it!
Now, there is no `pass` or Ellipsis where there is already a statement

Reply:
Merge it now